src/include/cell.i

```diff
@@ -143,6 +149,8 @@ struct __wt_cell_unpack {
 
 	uint64_t v;			/* RLE count or recno */
 
+	wt_timestamp_t start, stop;	/* start/stop timestamps */
```

Comment:
I'd prefer if the names had `timestamp` in them. When @michaelcahill first added the timestamp changes I pushed to use the full word rather than `ts`, but we've started moving towards `ts` and `TS` recently. Either would be fine with me - but I think we should move towards a consistent usage pattern.

Reply:
Will do. I'll go with `ts` so they aren't so long (start and stop aren't too bad, but `oldest_start` is getting longer).

Reply:
Pushed in 48f1f3a.